Modules/PID/src/TaskFT0TOF.cxx

```diff
@@ -50,23 +50,27 @@ namespace o2::quality_control_modules::pid
 
 bool MyFilter(const MyTrack& tr)
 {
-  return (tr.getP() < 2.0);
+  return (tr.getP() < 1.5);
```

Comment:
Maybe we want this configurable without having to change the code?

Reply:
Ciao @njacazio indeed thanks for spotting this! I changed it to configurable

Reply:
I guess that
float t0maxp = 1.5;
should be 
static float t0maxp;
(the same for the getter)

Reply:
Ciao @noferini thanks a lot for spotting this I changed it to static!